app/domain/logs.rb

```diff
@@ -9,6 +9,11 @@ module Conjur
       code: "CONJ00034I"
     )
 
+    FipsModeStatus = ::Util::TrackableLogMessageClass.new(
+      msg:  "Server set OpenSSL FIPS mode to {0}",
```

Comment:
Looking at the other log messages it seems that either
```
msg:  "OpenSSL FIPS mode set to {0}",
```
or
```
msg:  "Set OpenSSL FIPS mode to {0}",
```
would be more appropriate and consistent with other messages.

Is there a reason we need to explicitly call out the `Server`?

Reply:
@micahlee 
https://github.com/cyberark/conjur/pull/1653#discussion_r450121946
Same suggestion as Shuli :)

Reply:
Done